packages/forms/src/UIForm/fields/InputDateTimePicker/README.md

```diff
@@ -75,23 +76,8 @@ Example : "2018-01-01T10:35:48.951Z"
 		"autoFocus": false,
 		"disabled": false,
 		"placeholder": "Type the date here...",
-		"readOnly": false
+		"readOnly": false,
+		"validationMessage": "The date is not valid"
```

Comment:
As this is an example, and people will copy/paste it, can you improve the message to push good practices ?
Something like `The date format is not valid. Expected format: YYYY/MM/DD.`

Reply:
yep good point

Reply:
Update the behavior to have the message defined directly in the widget, as there could be only one.
In the goal to:
- enforce all applications to have the same message
- avoid having to add a new uiSchema property to use it properly
- reduce futur changes in uiSchemas when this error handling will evolve to a more friendly one and the unique message will not be needed